src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_nodejs/nodejs_translate.v2.json.baseline

```diff
@@ -14,38 +14,24 @@
 var google = require('googleapis');
 var translate = google.translate('v2');
 
-authorize(function(authClient) {
-  var request = {
-    // The text to detect
-    q: [],  // TODO: Update placeholder value.
-
-    auth: authClient
-  };
-
-  translate.detections.list(request, function(err, response) {
-    if (err) {
-      console.log(err);
-      return;
-    }
-
-    // TODO: Change code below to process the `response` object:
-    console.log(JSON.stringify(response, null, 2));
-  });
-});
+var apiKey = ''; // TODO: Update placeholder with desired API key.
+
+var request = {
+  // The text to detect
+  q: [],  // TODO: Update placeholder value.
+
+  auth: apiKey
+};
 
-function authorize(callback) {
-  google.auth.getApplicationDefault(function(err, authClient)) {
-    if (err) {
-      console.log('authentication failed: ', err);
-      return;
-    }
-    if (authClient.createScopedRequired && authClient.createScopedRequired()) {
-      var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
-      authClient = authClient.createScoped(scopes);
-    }
-    callback(authClient);
-  });
-}
+translate.detections.list(request, function(err, response) {
+  if (err) {
+    console.log(err);
+    return;
+  }
+
```

Comment:
Hmm, OK if you do it for now, and then we have a review with other language experts. My own bias is to keep the whitespace (I would in my code), but perhaps for a snippet the vertical space is too costly.